src/bandersnatch/configuration.py

```diff
@@ -1,14 +1,19 @@
 """
 Module containing classes to access the bandersnatch configuration file
 """
+from typing import Any
+from typing import Dict
+from typing import Optional
+from typing import Type
```

Comment:
Why not all one from?
i.e.
from typing import (Any, Dict, Optional, Type)

Reply:
personal preference though if you'd like a different style I will change it.

I use [reorder_python_imports](https://github.com/asottile/reorder_python_imports) -- the style here aims to reduce merge conflicts above everything else.

If you'd like, I could set you up with `isort` in this PR as well, that way we don't have to bicker about needless style